core/primitives/src/shard_layout.rs

```diff
@@ -757,28 +773,16 @@ impl ShardLayout {
     }
 }
 
-/// Maps an account to the shard that it belongs to given a shard_layout
-/// For V0, maps according to hash of account id
-/// For V1 and V2, accounts are divided to ranges, each range of account is mapped to a shard.
-///
-/// TODO(wacban) This would be nicer as a method in ShardLayout
+/// Maps an account to the shard that it belongs to given a shard layout.
+#[deprecated(note = "Please use `ShardLayout::account_id_to_shard_id` method instead")]
```

Comment:
Nice! Are there a lot of leftover usages? If there are none feel free to just remove this function.

Reply:
None left, I'll remove it then